app/controllers/dmptool/public_pages_controller.rb

```diff
@@ -55,10 +55,10 @@ def plan_index
 
       # Build the facets list and retain the user's current search, sort and faceting selections
       @facets = cached_facet_options
-      build_facet(facet: :funders, ids: selected_facets[:funder_ids], plans: plan_ids, facet_pos_in_plans: 1)
-      build_facet(facet: :institutions, ids: selected_facets[:org_ids], plans: plan_ids, facet_pos_in_plans: 2)
-      build_facet(facet: :languages, ids: selected_facets[:lang_ids], plans: plan_ids, facet_pos_in_plans: 3)
-      build_facet(facet: :subjects, ids: selected_facets[:sub_ids], plans: plan_ids, facet_pos_in_plans: 4)
+      build_facet(facet: :funders, ids: selected_facets[:funder_ids], plans: plan_ids, facet_pos_in_plans: 2)
+      build_facet(facet: :institutions, ids: selected_facets[:org_ids], plans: plan_ids, facet_pos_in_plans: 3)
+      build_facet(facet: :languages, ids: selected_facets[:lang_ids], plans: plan_ids, facet_pos_in_plans: 6)
```

Comment:
@briri - just curious why facet positions had to be changed from "3" and "4" to "6" & "7"

Reply:
it corresponds to the position of that id within the plan_id array